src/cares_wrap.cc

```diff
@@ -330,10 +323,10 @@ class QueryWrap : public AsyncWrap {
     Context::Scope context_scope(env()->context());
     Local<Value> arg;
     switch (status) {
-#define V(code)                                                               \
-      case ARES_ ## code:                                                     \
```

Comment:
It's unfortunate that the rule changes indentation here because previously it lined up well with the `switch()`. I'm guessing there is no way to change the behavior for specific situations like this...